documentation/Use Case Scenarios/Registering Account Scenario.txt

```diff
@@ -0,0 +1,24 @@
+Context: the user is a first time user
+Primary Flow:
+1.	The user loads the application
+2.	The user is taken to the login page
+3.	The user selects "register"
+4.	The user is taken to a registration page
```

Comment:
We may or may not have to update this after our discussion, depending on what @bfreder1 wants to do.